.travis.yml

```diff
@@ -1,3 +1,11 @@
 language: node_js
+sudo: false
```

Comment:
You shouldn't need to add `sudo: false` anymore as new projects always run on the new Travis infrastructure. Did you have any issues with it?